src/validator/CreditCardVendorsReader.java

```diff
@@ -0,0 +1,58 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class CreditCardVendorsReader implements CreditCardVendorReading {
```

Comment:
As I've mentioned before, I'd refactor this class to use Spring properties resolution if we have to keep them in a file, or maybe use a @Repository annotation if we plan to turn this configuration to dynamic (Stored in a database)